src/CalcViewModel/Common/LocalizationSettings.h

```diff
@@ -176,56 +174,39 @@ namespace CalculatorApp
                     return localizedString;
                 }
 
-                size_t i = 0;
-                auto localizedStringData = localizedString->Data();
-                size_t length = localizedString->Length();
-                std::unique_ptr<wchar_t[]> englishString(new wchar_t[length + 1]); // +1 for the null termination
+                std::wstring englishString;
+                englishString.reserve(localizedString->Length());
```

Comment:
very minor: can we use a stringstream instead?

Reply:
I don't mind making this change, but since C++ isn't the language I use for my day job, I'm curious why a std::wstringstream is the better choice in this situation.  I put together a simple function to remove spaces on compiler explorer and clang is able to generate way less assembly using std::wstring over std::wstringstream.
https://godbolt.org/z/PjXGFW